identity/credentials_oidc.go

```diff
@@ -30,7 +30,7 @@ type CredentialsOIDCProvider struct {
 	InitialAccessToken  string `json:"initial_access_token"`
 	InitialRefreshToken string `json:"initial_refresh_token"`
 	Organization        string `json:"organization,omitempty"`
-	UseAutoLink         bool   `json:"use_auto_link,omitzero"`
+	UseAutoLink         bool   `json:"use_auto_link,omitempty"`
```

Comment:
This works the same, as `omitzero` for booleans, so also fine. But omitzero is a valid tag in Go 1.24, which Kratos is updated to.